### PR TITLE
Update pooling.py

### LINKED ISSUE #46998: Documentation mistake in tf.keras.layers.MaxPool2D python section
## URL(s) with the issue: 
https://www.tensorflow.org/api_docs/python/tf/keras/layers/MaxPool2D

## Description of issue (what needs changing):

A Documentation mistake in tf.keras.layers.MaxPool2D python section(example)

### Clear description

There is a documentation mistake in tf.keras.layers.MaxPool2D python section(example). 
In the description part of there are few examples given with their respective code snippets.
Under this text-"For example, for stride=(2,2) and padding="valid":" in the code snippet of example instead of **strides=(2,2)**, **strides=(1,1)** is written.

Screenshot 2021-02-08 at 10 16 05 AM

### PR DESCRIPTION
**Pull request by amahendrakar**
Updated the value of strides argument in example snippet.

Fixes the change as suggested in #46998.

### CODE REVIEW

File: tensorflow/python/keras/layers/pooling.py [hunk 1]
```diff
@@ -366,7 +366,7 @@ class MaxPooling2D(Pooling2D):
   ...                  [9., 10., 11., 12.]])
   >>> x = tf.reshape(x, [1, 3, 4, 1])
   >>> max_pool_2d = tf.keras.layers.MaxPooling2D(pool_size=(2, 2),
-  ...    strides=(1, 1), padding='valid')
+  ...    strides=(2, 2), padding='valid')
   >>> max_pool_2d(x)
   <tf.Tensor: shape=(1, 2, 3, 1), dtype=float32, numpy=
```
Comment (qlzh727) [inline tensorflow/python/keras/layers/pooling.py:371]: Thanks for the update. The output tensor shape and value need to be updated as well.

>>> max_pool_2d = tf.keras.layers.MaxPooling2D(pool_size=(2, 2), strides=(2, 2), padding='valid')
>>> max_pool_2d(x)
<tf.Tensor: shape=(1, 1, 2, 1), dtype=float32, numpy=
array([[[[6.],
         [8.]]]], dtype=float32)>

Comment (amahendrakar) [reply tensorflow/python/keras/layers/pooling.py]: Updated the shape and output. Thanks for pointing it out.

**Review by qlzh727 — APPROVED** [review]

**Review by gbaned — APPROVED** [review]
